.github/workflows/community_ci.yml

```diff
@@ -76,6 +76,60 @@ jobs:
           sentry_dsn: ${{ secrets.SENTRY_AIRBYTE_CI_DSN }}
           subcommand: "format check all"
           is_fork: "true"
+
+  check-review-requirements:
+    name: Check if a review is required from Connector teams on fork
+    if: github.event.pull_request.head.repo.fork == true
+    environment: community-ci-auto
+    runs-on: community-tooling-test-small
+    needs: fail_on_protected_path_changes
+    timeout-minutes: 10
+    env:
+      MAIN_BRANCH_NAME: "master"
+    permissions:
+      pull-requests: write
```

Comment:
Based on [docs](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-pull-requests), I think we need this in order to add the requested reviewers. Do we also need `statuses: write`? It seemed to me like maybe that was for the workflows which kick off multiple status events (connector tests and poe tests)